openwisp_users/apps.py

```diff
@@ -87,6 +87,9 @@ def set_default_settings(self):
             }
             setattr(settings, 'SWAGGER_SETTINGS', SWAGGER_SETTINGS)
 
+        ACCOUNT_ADAPTER = "openwisp_users.accounts.adapter.EmailAdapter"
```

Comment:
let's do as `LOGIN_URL` and `LOGOUT_URL`, set this only if not present in settings (we allow users to customize it if needed).

Please ensure quoting style is consistent (single quotes).